frontend/test/playwright/utils/page.ts

```diff
@@ -4,18 +4,6 @@
 
 import type { Page } from "@playwright/test"
 
-export const removeHiddenOverflow = async (page: Page) => {
```

Comment:
We no longer have `embedded` mode, and this function was not used for a long time